rust-workspace/crates/ntc-vault-cli/src/identity_files.rs

```diff
@@ -0,0 +1,45 @@
+//! Support for working with identity files.
+
+use std::path::{Path, PathBuf};
+
+use anyhow::Context;
+use confy::ConfyError;
+use ntc_data_packages::identity::VaultIdentity;
+use rusty_sodalite::types::SecureSeed;
+use serde::{Deserialize, Serialize};
+use serde_with::base64::Base64;
+use serde_with::serde_as;
+
+#[derive(Default, Debug)] // core
+#[serde_as]
+#[derive(Serialize, Deserialize)] // serde
+pub(crate) struct VaultIdentityConfig {
+    pub(crate) name: String,
+
+    #[serde_as(as = "Base64")]
+    pub(crate) seed: SecureSeed,
+}
+
+impl VaultIdentityConfig {
+    pub(crate) fn load(path: impl AsRef<Path>) -> anyhow::Result<Self> {
+        let path = path.as_ref();
+        confy::load_path(path).with_context(|| format!("Failed to load {}", path.to_string_lossy()))
+    }
+
+    pub(crate) fn store(&self, path: impl AsRef<Path>) -> anyhow::Result<()> {
+        let path = path.as_ref();
+        confy::store_path(path, self)
+            .with_context(|| format!("Failed to store {}", path.to_string_lossy()))
+    }
+
+    pub(crate) fn get_default_path() -> Result<PathBuf, ConfyError> {
+        confy::get_configuration_file_path("ntc-vault", "identity")
```

Comment:
When trying to create a new identity, line 31 crashes:
```
/home/bill/.cargo/bin/cargo +nightly run --color=always -- identity create -n Bill
...
Error: Failed to store /home/bill/.config/ntc-vault/identity.toml

Caused by:
    0: Failed to open configuration file
    1: No such file or directory (os error 2)

Stack backtrace:
   ...
   4: ntc_vault_cli::identity_files::VaultIdentityConfig::store
             at ./src/identity_files.rs:31:9
```
We think this is as a result of line 36 `get_configuration_file_path`

Reply:
Great catch!

This wasn't triggering locally because I started out with `store`, which created the `ntc-vault` config subdirectory, and then switched to using `store_path` while the config subdirectory was already created.

I filed an upstream fix here: https://github.com/rust-cli/confy/pull/60